src/components/LocationPage/ChartsHolder.tsx

```diff
@@ -143,16 +169,28 @@ const ChartsHolder = React.memo(({ region, chartId }: ChartsHolderProps) => {
     scrollTo(shareBlockRef.current, -352);
   }, []);
 
+  const [clickedStatMetric, setClickedStatMetric] = useState<Metric | null>(
+    null,
+  );
+
   const onClickMetric = useCallback(
     (metric: Metric) => {
       trackEvent(
         EventCategory.METRICS,
         EventAction.CLICK,
         `Location Header Stats: ${Metric[metric]}`,
       );
-      scrollTo(metricRefs[metric].current);
+      setClickedStatMetric(metric);
+      const groupWithMetric = getChartGroupFromMetric(metric);
+      const chartBlockRef = groupWithMetric
+        ? chartBlockRefs[groupWithMetric.groupHeader]
+        : null;
+      if (chartBlockRef?.current) {
+        setScrolledWithRef(true);
```

Comment:
So I was trying to remember what this scrolledWithRef thing was for, and I think:
1. It was for when you scroll to the chart via URL, (e.g. https://covidactnow.org/us/washington-wa/chart/5/) and so we don't need to set it to true here (though it probably doesn't hurt).  We might want to rename it to `scrolledWithUrl` or something.
2. It looks like scrolling via URL like https://covidactnow.org/us/washington-wa/chart/5/ is currently broken which means our share links are broken right now (even in prod)... I opened a trello card: https://trello.com/c/K3JG32kp/1431-chart-share-urls-dont-scroll-to-chart-anymore